recipe/meta.yaml

```diff
@@ -32,11 +32,13 @@ requirements:
     - libtool  # [linux]
     - make  # [linux]
     - pkg-config  # [linux]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
     - python
   run:
-    - importlib-metadata >=2.0,<3  # [py<38]
+    - importlib-metadata >=2.0,<3
```

Comment:
```suggestion
```

Reply:
i already removed it

Reply:
no i meant remove the whole line. it's not necessary with python >=3.8